src/reformat.c

```diff
@@ -30,12 +30,20 @@ static avifBool avifPrepareReformatState(const avifImage * image, const avifRGBI
     //
     // YCgCo performs limited-full range adjustment on R,G,B but the current implementation performs range adjustment
     // on Y,U,V. So YCgCo with limited range is unsupported.
+#if defined(AVIF_ENABLE_EXPERIMENTAL_YCGCO_R)
+    const avifBool useYCgCoR = (image->matrixCoefficients == AVIF_MATRIX_COEFFICIENTS_YCGCO_R);
+    const avifMatrixCoefficients lastMC = AVIF_MATRIX_COEFFICIENTS_YCGCO_R;
```

Comment:
Nit: It may be better to add an `AVIF_MATRIX_COEFFICIENTS_LAST` or `AVIF_MATRIX_COEFFICIENTS_MAX` value to the enum type in avif.h.